src/main/scala/uk/gov/hmrc/supportfrontend/views/Utils.scala

```diff
@@ -46,20 +48,23 @@ trait Utils {
     htmlArgs.padLeft(if (attributes.nonEmpty) padCount else 0)
   }
 
-  object NonEmptyString {
-    def unapply(s: String): Option[String] =
-      if (s != null && s.nonEmpty) Some(s) else None
-  }
-
-  private[views] def isNonEmptyOptionString(value: Option[String]) = value match {
+  def isNonEmptyOptionString(value: Option[String]): Boolean = value match {
     case Some(NonEmptyString(_)) => true
     case _                       => false
   }
 
-  private[views] def calculateAssetPath(path: Option[String], file: String): String =
+  def calculateAssetPath(path: Option[String], file: String): String =
```

Comment:
Is there a reason to make this public?

Reply:
This will no longer work because the package structure has changed. Previously this method was private to the package `uk.gov.hmrc.govukfrontend.views`. However, because this method is now in the package `uk.gov.hmrc.supportfrontend`, it cannot be made private to a different package of `uk.gov.hmrc.govukfrontend` (see this post on SO: https://stackoverflow.com/questions/9032631/package-private-modifier-in-scala-2-8

Reply:
calculateAssetPath is really private to GovukTemplate and govukfrontend - it's not used by the hmrc components at all. Can it privately live in govukfrontend.components?